src/helpers/promptUtils.ts

```diff
@@ -0,0 +1,69 @@
+import { Question, Questions, ux } from '@cto.ai/sdk'
+
+//This re-prompts user until their answer passes validation
+//validate takes in the return value of ux.prompt, and forces re-prompt if it returns false
+//errorMessage is displayed to user when validation fails
+export const validatedPrompt = async (
+  prompt: Question | Questions,
+  validate: (response: any) => boolean,
+  errorMessage: string,
+) => {
+  let response: any
+  let repeatedPrompt = false
+  //keep asking as long as validation fails
+  do {
+    if (repeatedPrompt) {
+      //not our first prompt
+      //must have errored, so print the error message
+      await ux.print(errorMessage)
+    }
+    response = await ux.prompt(prompt)
+    repeatedPrompt = true
+  } while (!validate(response))
+  return response
+}
+
+//this replicates the SDK 1.x functionality,
```

Comment:
Let's clean up this function documentation, make it consistent with the others, clarify inputs and output and 1/2 lines for what it does. Also, leverage types..

Reply:
Cleaned up.